File: Python/Colleen.py
#!/usr/bin/env python3
'''
	Self Replication
'''

def	oussama():
	code = "#!/usr/bin/env python3\n'''\n\tSelf Replication\n'''\n\ndef\toussama():\n\tcode = {0}\n\treturn code\n\nprint(oussama().format(repr(oussama())))"
	return code

print(oussama().format(repr(oussama())))

File: JavaScript/Colleen.js
#!/usr/bin/env node

/*
	Self Replication
*/

oussama = () => { return "#!/usr/bin/env node\n\n/*\n\tSelf Replication\n*/\n\noussama = %s \n\n(function main() {\n\t/*\n\t\tQuine is the Queen\n\t*/\n\tconsole.log(oussama(), oussama.toString())\n})()\n" } 

(function main() {
	/*
		Quine is the Queen
	*/
	console.log(oussama(), oussama.toString())
})()

